config/default_records/submission_templates/009_scrn_core_cell_extraction_submission_templates.yml

```diff
@@ -0,0 +1,12 @@
+# These submission templates are associated with the Limber scRNA Core Cell Extraction pipeline.
+# They are in a separate file to allow us to soft-release them before
+# the real release.
+---
+Limber - scRNA Core Cell Extraction:
+  submission_class_name: "LinearSubmission"
+  related_records:
+    request_type_keys: ["limber_scrna_core_cell_extraction"]
+    project_name: "Project scRNA Core"
```

Comment:
NB. projects and studies with these names must exist in the various environments. Usually it is the job of the SSRs to create them. Do these exist in Production (and in UAT - if not we can create those ones manually via UAT actions)?

Reply:
@StephenHulme and I discussed this one. I think what we want is to *not* specify the Project or Study at all here, and use the 'autodetect_studies_projects' feature in the API to choose the appropriate ones to put the requests under. (Unlike in Cardinal)